CHANGES.txt

```diff
@@ -14,6 +14,16 @@ Change History
 
 - Fix or ignore some PyCharm inspection warnings.
 
+- Use generic SQLAlchemy type Text as base type for JsonType. This allows
+  SQLAlchemy to map Text type to the most suitable type available on given
+  database system. Previously used TEXT type is not available in Oracle
+  database. In case of existing installation of Kotti with database system,
+  for which SQLAlchemy maps generic Text type to type different than TEXT it's
+  necessary to either convert existing columns "nodes._acl" and
+  "nodes.annotations" to that type or configure SQLAlchemy to map generic Text
```

Comment:
Converting the existing columns sounds fine. Can we write a migration that does that?